workspaces/libnpmpublish/lib/publish.js

```diff
@@ -62,17 +62,18 @@ Remove the 'private' field from the package.json to publish it.`),
   return res
 }
 
-const patchManifest = (_manifest, opts) => {
+const patchManifest = async (_manifest, opts) => {
   const { npmVersion } = opts
-  // we only update top-level fields, so a shallow clone is fine
-  const manifest = { ..._manifest }
-
-  manifest._nodeVersion = process.versions.node
-  if (npmVersion) {
-    manifest._npmVersion = npmVersion
+  const steps = PackageJson.normalizeSteps.filter(s => s === 'fixname')
```

Comment:
```suggestion
  const steps = ['fixName']
```

`fixName` wasn't added to the normalizeSteps.  Even if it was there's no need to do anything but hard set this.

Also it's case sensitive.